model/events/parse.go

```diff
@@ -4,6 +4,7 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/onflow/flow-go/fvm/evm/types"
 	"github.com/onflow/flow-go/model/flow"
 )
```

Comment:
hmm, this looks like a dependency that shouldn't happen.

Reply:
Mainly added so that we re-use the `"evm"` constant, but I can remove it. `"flow"` is also matched against a string literal.